packages/product-feed-google/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Changed
+- visibility of all private properties and methods of DataFactories was changed to protected (@Miroslav-Stopka)
```

Comment:
Please why do You update  product-feed-* changelogs when there is nothing changed?

Reply:
https://github.com/shopsys/shopsys/pull/116/files#diff-eb626cdf8d2de8fa06580571c3ffc220